package.json

```diff
@@ -43,6 +44,7 @@
     "onoff": "^1.1.2",
     "openzwave-shared": "^1.3.2",
     "serialport": "^4.0.7",
+    "speaktome-web": "https://github.com/mozilla/speaktome-web",
```

Comment:
Are you eventually going to publish this on npm?

Reply:
Yes, we published already. Am fixing this.

Reply:
The module is here btw https://www.npmjs.com/package/speaktome-api